src/main/java/com/revature/studyforce/flashcard/dto/FlashcardAllDTO.java

```diff
@@ -0,0 +1,40 @@
+package com.revature.studyforce.flashcard.dto;
+
+import com.revature.studyforce.flashcard.model.Flashcard;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.util.Assert;
+import java.util.function.Function;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlashcardAllDTO {
+    private int flashcardId;
+    private int creatorId;
+    private String topicName;
+    private String question;
+    private int difficulty;
+    private boolean isResolved;
+
+    /**
+     * Converts Flashcard to FlashcardDTO
+     * @return - FlashcardDTO
+     */
+    public static Function<Flashcard, FlashcardAllDTO> convertToDTO() {
```

Comment:
incorrect documentation, this method does not convert a flashcard to DTO nor does it return a flashcardDTO. This method acts as a getter which returns a function which itself can be used to convert a flashcard to a FlashcardAllDTO